### PR TITLE
Glridb 507 publication list is broken

### PR DESCRIPTION
**Pull request by ee-usgs**
Please merge GLRIDB-510 (Previous pull Req) first, so that those changes do not show up here.
See comments for the March 14 commit for the actual changes.

### CODE REVIEW

File: glri-catalog-ui/src/main/webapp/js/app/newProjectForm.js [hunk 1]
```diff
@@ -167,6 +168,39 @@ function($scope, $http, Status, ScienceBase) {
...
+				var model = modelBinding.split('.');
+				var value = $scope[model[0]][model[1]];
+				
+				if (value != undefined && value.length != 0) {
+					var response = parseAndRemoveOneUrl(value);
+					var msg = undefined;
+					
+					if (response.isOk) {
+						if (response.value == undefined) {
+							msg = "No url was found in this field";
```
Comment (duselmann) [inline glri-catalog-ui/src/main/webapp/js/app/newProjectForm.js]: This looks like a 'required' clause but I do not see how you are checking required-ness. Are all URL fields going to be required? If so then this is OK.

**Comment by duselmann** [comment]
Feel free to merge after considering the one comment.

**Comment by duselmann** [comment]
I thought this all looked very familiar and I was correct. Most of this seems to be from #247 . If that needs to be merged as well then it is ready. Otherwise, we could close it in favor of this one.

File: glri-catalog-ui/src/main/webapp/js/app/sciencebase.js [hunk 2]
```diff
@@ -436,7 +436,7 @@ function($http, Status, FocusAreaManager, $rootScope){
 	
 	ctx.buildUrl = function(resource) {
 		var url = Status.CONST.BASE_QUERY_URL+ "resource="+encodeURI(resource+"&")
-			+"fields=" +encodeURI("url,summary,tags,title,contacts,hasChildren,webLinks,purpose,body,dateCreated,parentId,facets,dates");
```
Comment (ee-usgs) [inline glri-catalog-ui/src/main/webapp/js/app/sciencebase.js]: This is the only relevant change.  All others can via the PR 247, which should go away when that one is merged.

**Pull request merged**